weather-map-wk7/client/src/components/weatherCard.js

```diff
@@ -0,0 +1,42 @@
+const WeatherCard = ({data}) => {
+
```

Comment:
This might be a good place to experiment with object destructuring assignment:
`const { main, name, sys, weather, wind } = data;`